pkg/util/provider/machinecontroller/machine_test.go

```diff
@@ -123,82 +123,6 @@ var _ = Describe("machine", func() {
 		)
 	})
 
-	/*
-		Describe("##updateMachineConditions", func() {
-			Describe("Update conditions of a non-existing machine", func() {
-				It("should return error", func() {
-					stop := make(chan struct{})
-					defer close(stop)
-
-					objects := []runtime.Object{}
-					c, trackers := createController(stop, testNamespace, objects, nil, nil)
-					defer trackers.Stop()
-
-					testMachine := &v1alpha1.Machine{
```

Comment:
I think we shouldn't remove these commented codes until we are sure we have addressed whatever they suggest.

Reply:
ok